scripts/install

```diff
@@ -72,14 +24,6 @@ then
 	echo "$bootstrap_peers" | grep -E '[0-9a-f]{64}@((\b25[0-5]|\b2[0-4][0-9]|\b[01]?[0-9][0-9]?)(\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3}|([a-zA-Z0-9][a-zA-Z0-9-]{0,61}[a-zA-Z0-9]\.)+[a-zA-Z]{2,}):[0-9]{1,4}' || ynh_die --message="friend server id must have id with the following form : 1799bccfd7411eddcf9ebd316bc1f5287ad12a68094e1c6ac6abde7e6feae1ec@192.168.1.1:1234 or  1799bccfd7411eddcf9ebd316bc1f5287ad12a68094e1c6ac6abde7e6feae1ec@example.tld:1234"
 fi
 
-if [ "$datadir" = "/home/yunohost.app/__APP_NAME__/data" ]
-then
-	datadir="/home/yunohost.app/$app/data"
-fi
-
-# Register (book) web path
-ynh_webpath_register --app=$app --domain=$domain --path_url="/"
-
 #=================================================
 # LOOKING FOR VIRTUALISATION
```

Comment:
The whole thing about virtualisation is a bit confusing ... I'm confused as to what is checked exactly ... like `which modprobe ` .... uuuuh wut ...

If the point is to check wether or not we are in a container or virtualized system, I would recommend using `systemd-detect-virt` which should be much more reliable

And I would also encourage not storing this as a setting, but defining it on-the-fly in _common.sh, because you could imagine scenarios in which this app gets backup / restored on different hardware etc ...

Reply:
For the record, I understand nothing of this, so I can't help on this - but maybe that could be a separate second step to reduce the changes in one leap ?

> And I would also encourage not storing this as a setting, but defining it on-the-fly in _common.sh, because you could imagine scenarios in which this app gets backup / restored on different hardware etc ...

Great advice :+1: